.devcontainer/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM fdev321123/arc-devcontainer:0.0.1
```

Comment:
what is this container?

Reply:
The output of base.Dockerfile

I'll add a makefile to build and push it